x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/__mocks__/overview_logic.mock.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IOverviewValues } from '../overview_logic';
+import { IAccount, IOrganization, IUser } from '../../../types';
+
+export const mockLogicValues = {
+  accountsCount: 0,
+  activityFeed: [],
+  canCreateContentSources: false,
+  canCreateInvitations: false,
+  currentUser: {} as IUser,
+  fpAccount: {} as IAccount,
+  hasOrgSources: false,
+  hasUsers: false,
+  isFederatedAuth: true,
+  isOldAccount: false,
+  organization: {} as IOrganization,
+  pendingInvitationsCount: 0,
+  personalSourcesCount: 0,
+  sourcesCount: 0,
+  dataLoading: true,
+  hideOnboarding: false,
+  hasErrorConnecting: false,
+  statsColumns: 'halves',
+  flashMessages: {},
+} as IOverviewValues;
+
+export const mockLogicActions = {
+  initializeOverview: jest.fn(() => ({})),
+};
+
+jest.mock('kea', () => ({
+  ...(jest.requireActual('kea') as object),
+  useActions: jest.fn(() => ({ ...mockLogicActions })),
+  useValues: jest.fn(() => ({ ...mockLogicValues })),
+}));
+
+import { useValues } from 'kea';
+
+export const setMockValues = (values: object) => {
+  (useValues as jest.Mock).mockImplementationOnce(() => {
+    return {
+      ...useValues(),
+      ...values,
+    };
+  });
+};
```

Comment:
`...useValues()` was confusing to me at first until I realized it was just calling the already-mocked `mockLogicValues`. What do you think about making this more explicit in what we're returning? 🤷 

```suggestion
export const setMockValues = (values: object) => {
  (useValues as jest.Mock).mockImplementationOnce(() => ({
    ...mockLogicValues,
    ...values,
  }));
};
```

Reply:
I'm also wondering if there's any way to DRY `setMockValues` out so that multiple logic files/components can reuse it, in which case maybe `useValues()` is more correct (but relies upon useValues already having been mocked?) 🤔 Sorry, just thinking out loud now

Reply:
Yeah that was my thinking when I ported this over from Jason's code; my hope is to DRY this out.